sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamReader.scala

```diff
@@ -249,6 +249,8 @@ final class DataStreamReader private[sql](sparkSession: SparkSession) extends Lo
    * `com.databricks.spark.csv`.</li>
    * <li>`escape` (default `\`): sets the single character used for escaping quotes inside
    * an already quoted value.</li>
+   * <li>`escapeQuoteEscaping` (default `\0`): sets the single character used for escaping
+   * the quote-escape character.</li>
```

Comment:
The option name is not clear to me. Any better name? 

`\0` is the NUL terminator?

Reply:
Why not following the CSV API docs?

http://docs.univocity.com/parsers/1.5.1/com/univocity/parsers/csv/CsvFormat.html

> charToEscapeQuoteEscaping (defaults to '\0' - undefined): character used for escaping the escape for the quote character
> e.g. if the quoteEscape and charToEscapeQuoteEscaping are set to '\', the value " \\\" a , b \\\" " is parsed as [ \" a , b \" ]

Reply:
About the naming, Univocity and spark uses little bit different option namings as follows:

| univocity | spark |
| :--            | :--       |
| delimiter        | sep          |
| quote             | quote          |
| quoteEscape  |  escape         |
| charToEscapeQuoteEscaping |  escapeQuoteEscaping         |

I tried to follow Spark's way by simply removing `charTo`.